roles/openshift_health_checker/openshift_checks/diskspace_availability.py

```diff
@@ -0,0 +1,44 @@
+# pylint: disable=missing-docstring
+from openshift_checks import OpenShiftCheck, get_var
+from openshift_checks.mixins import NotContainerizedMixin
```

Comment:
This seems to be a leftover, unused import.
Unlike Go, the Python interpreter doesn't complain, but I'm sure this is failing our CI.